app/app.go

```diff
@@ -428,6 +442,12 @@ func NewApp(
 	bApp.SetInterfaceRegistry(interfaceRegistry)
 	bApp.SetTxEncoder(txConfig.TxEncoder())
 
+	// for original AuctionFactory
+	// maxTx can be changed (0 is unlimited)
+	factory := mempool.NewDefaultAuctionFactory(txConfig.TxDecoder())
+	mempool := mempool.NewAuctionMempool(txConfig.TxDecoder(), txConfig.TxEncoder(), 0, factory)
```

Comment:
nit: would suggest making maxTx an explicit constant, rather than setting to zero inline. Setting as constant improves readability / is easier to change long-term